src/js/controller/DrawingController.js

```diff
@@ -140,6 +142,12 @@
     $.publish(Events.MOUSE_EVENT, [event, this]);
     var frame = this.piskelController.getCurrentFrame();
     var coords = this.getSpriteCoordinates(event.clientX, event.clientY);
+    if (typeof event.targetTouches != 'undefined' &&
+        typeof event.targetTouches[0] != 'undefined' &&
+        typeof event.targetTouches[0].pageX != 'undefined' &&
+        typeof event.targetTouches[0].pageY != 'undefined') {
```

Comment:
(just a question, no need to change anything here) 

Is there a good reason to prefer pageX/pageY to clientX/clientY here ? (In theory for Piskel we clientX/Y and pageX/Y should be equivalent because we have no scrollbar right ?)